src/ctia/lib/riemann.clj

```diff
@@ -129,33 +129,36 @@
 ;; based on riemann-reporter.core/wrap-request-metrics
 (defn wrap-request-logs
   "Middleware to log all incoming connections to Riemann"
-  [handler service-name]
-  (let [config (get-in @prop/properties [:ctia :log :riemann])
-        _ (log/info "Riemann request logging initialization")
+  [handler metric-description]
+  {:pre [(string? metric-description)
+         (seq metric-description)]}
```

Comment:
Why no `s/defn`?

Related, lately I'm enjoying `s/pred` for accomplishing stronger assertions (like `seq` here) while not deviating from Schema and also not losing some informative reporting (which :pre lacks by default)